src/ray/raylet/scheduling/cluster_task_manager_test.cc

```diff
@@ -1193,7 +1193,7 @@ TEST_F(ClusterTaskManagerTest, HeartbeatTest) {
         // infeasible, ready, CPU, GPU, size
         {1, 0, 10, 1, 2},
         {1, 0, 9, 5, 2},
-        {0, 1, 1, 0, 1}};
+        {0, 0, 1, 0, 1}};
```

Comment:
This is such that we are no longer treating task that has resources allocated in the "num_ready"  demand.